server/tests/src/test/java/org/infinispan/server/security/authorization/AbstractAuthorization.java

```diff
@@ -263,29 +264,50 @@ public void testRestNonAdminsMustNotModifyLoggers() {
    }
 
    @Test
-   public void testRestNonAdminsMustNotObtainReport() {
+   public void testRestAdminsShoudleBeAbleToAdminServer() {
+      RestClientConfigurationBuilder adminConfig = restBuilders.get("admin");
+      assertEquals(204, sync(getServerTest().rest().withClientConfiguration(adminConfig).get().server().connectorStop("endpoint-alternate-1")).getStatus());
+      assertEquals(204, sync(getServerTest().rest().withClientConfiguration(adminConfig).get().server().connectorStart("endpoint-alternate-1")).getStatus());
```

Comment:
nit, all this asserts could be in the single method:

```
private void checkAdminServer(int status, String user) {
  RestClientConfigurationBuilder adminConfig = restBuilders.get(user);
  assertEquals(status, ....);
  ... 
}

Reply:
ps. maybe ^ can be done in another PR. the ignore cache tests aren't checking if the admin can do the task or not (same for xsite as reported by Katia).
ps2. `org.infinispan.server.test.core.Common#assertStatus()` could be used instead of `asserEquals()`  :)

Reply:
Yes, let's defer to my current authz work